jest.config.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
     '^~cpn/(.*)$': '<rootDir>/apps/mochi-web/components/$1',
     '^~(.*)$': '<rootDir>/apps/mochi-web/$1',
   },
-  transformIgnorePatterns: ['[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$'],
+  transformIgnorePatterns: [`node_modules/(?!(?:.pnpm/)?(flat))`],
```

Comment:
Why do we need to make the change?

Reply:
Jest has strange behavior when running tests for the button component. 
It traverses through `theme/src/util.ts`, where `theme/src/util.ts` use the `flat` module, which contains some ES6 code that Jest cannot compile, resulting in an error. 
Adding this line will instruct Jest to transform the `flat` module before running the test.